src/shared/components/CloudUpgradeButton.tsx

```diff
@@ -44,11 +47,23 @@ const CloudUpgradeButton: FC<StateProps & OwnProps> = ({
     [`${className}`]: className,
   })
 
+  const [iconVariant, setIconVariant] = useState<IconFont>(IconFont.Upgrade)
```

Comment:
i see what you're going for here, but the name `iconVariant` makes it seem like it is *the* variant, not one of the variants.

what about `iconVariantForUser` or `usersIconVariant` to make it the specific variant for this user?